aytests/lvm.xml

```diff
@@ -140,11 +140,6 @@ mv /var/run/zypp.sav /var/run/zypp.pid
     </remove-packages>
   </software>
 
-  <language>
```

Comment:
As per our discussion, I've moved this check to lvm_auto.list test, where we also set German as system language.